demo/meeting/src/containers/AppCheckerResult.tsx

```diff
@@ -0,0 +1,287 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useEffect, useState } from 'react';
+import {
+  RadioGroup,
+  Flex,
+  Badge,
+  useAppCheckerManager
+} from 'amazon-chime-sdk-component-library-react';
+
+import { endMeeting } from '../utils/api';
+import { useAppState } from '../providers/AppStateProvider';
+import Spinner from '../components/Spinner';
+import { Table, TableHead, TableRow, TableCell } from '../components/Table';
+
+const videoQualities = [
+  { width: 640, height: 480 },
+  { width: 1280, height: 720 },
+  { width: 1920, height: 1080 }
+];
+
+const options = [
+  { value: 'Succeeded', label: 'Yes' },
+  { value: 'Failed', label: 'No' }
+];
+
+const AppCheckerResult = () => {
```

Comment:
We need to break these down into individual consumable components and hooks so that developers can easily use them by dropping in the components or creating their own with hooks. Right now we are just exposing a thin wrapper around the SDK AppChecker and not really adding much additional value